exercises/bowling/src/Bowling.hs

```diff
@@ -0,0 +1,14 @@
+module Bowling (bowlingStart, roll, score) where
+
+-- modify this appropriately
+data Bowling = Bowling
```

Comment:
Since #302 , the stub solutions do not have comments anymore.

We did this because most people where submitting solutions without removing them, and `HINTS.md` seemed the most appropriate place, so people will have to get used to reading the `README.md` file in case of trouble.

Also, to mark data types that need to be changed by the user, we are using `Dummy` and adding instructions in `HINTS.md`. Take a look at the exercises `forth`, `custom-set` and `binary-search-tree` that we fixed; all of them now have a `HINTS.md` and a data type to be implemented.

We are trying to make the stub solutions allow compiling and running the tests when that doesn't ruin the exercise, so it would be nice if this exercise could allow that from the start #421.